### PR TITLE
Translate  `en/model_doc` to JP

### LINKED ISSUE #27556: [i18n-JP] Translating `en/model_doc` docs to Japanese
Hi!

Let's bring the documentation to all the Japanese-speaking community 🌐 

Who would want to translate? Please follow the 🤗 [TRANSLATING guide](https://github.com/huggingface/transformers/blob/main/docs/TRANSLATING.md). Here is a list of the files ready for translation. Let us know in this issue if you'd like to translate any, and we'll add your name to the list.

Some notes:

* Please translate using an informal tone (imagine you are talking with a friend about transformers 🤗).
* Please translate in a gender-neutral way.
* Add your translations to the folder called `ja` inside the [source folder](https://github.com/huggingface/transformers/tree/main/docs/source).
* Register your translation in `ja/_toctree.yml`; please follow the order of the [English version](https://github.com/huggingface/transformers/blob/main/docs/source/en/_toctree.yml).
* Once you're finished, open a pull request and tag this issue by including #issue-number in the description, where issue-number is the number of this issue. Please ping @stevhliu and @MKhalusova for review.
* 🙋 If you'd like others to help you with the translation, you can also post in the 🤗 [forums](https://discuss.huggingface.co/).

## Model_doc section
- [x]  [bark.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/bark.md) #27264
- [x]  [bart.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/bart.md) #27264
- [x]  [barthez.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/barthez.md) #27264
- [x]  [bartpho.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/bartpho.md) #27264
- [x]  [beit.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/beit.md) #27264
- [x]  [bert-generation.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/bert-generation.md) #27264
- [x]  [bert-japanese.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/bert-japanese.md) #27264
- [x]  [bert.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/bert.md) [](#27264)
- [x]  [bertweet.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/bertweet.md) #27264
- [x]  [big_bird.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/big_bird.md) #27264 
- [x]  [bigbird_pegasus.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/bigbird_pegasus.md) #27264
- [x]  [biogpt.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/biogpt.md) #27264
- [x]  [bit.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/bit.md) #27264
- [x]  [blenderbot-small.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/blenderbot-small.md) #27264
- [x]  [blenderbot.md](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/blenderbot.md) #27264

Keep on adding more as you go 🔥

### PR DESCRIPTION
**Pull request by rajveer43**
# What does this PR do?

translate files

Fixes #27556

## Who can review?

Documentation: @stevhliu and @MKhalusova

### CODE REVIEW

**Review by stevhliu — COMMENTED** [review]
Hi, thanks for your PR and apologies for the delay! Several things:

1. There is some overlap in models (`albert.md` to `autoformer.md`) here with #27401 so lets remove those from this PR. You should have some way of claiming models from the list in #27392 to avoid future duplicate work. 
2.  This is also slightly out-of-date with the recent standardization to the model API docs introduced in #26876. Would you please mind updating?
3. As you mentioned [here](https://github.com/huggingface/transformers/issues/27392#issuecomment-1803669747), try to work on ~10 at a time please. 🙏 
4. You'll need to add the docs you're working on to the `toctree` to get the `build pr` CI test to pass.

**Comment by rajveer43** [comment]
Sure

> Hi, thanks for your PR and apologies for the delay! Several things:
> 
> 1. There is some overlap in models (`albert.md` to `autoformer.md`) here with [Translating `en/model_doc` docs to Japanese. #27401](https://github.com/huggingface/transformers/pull/27401) so lets remove those from this PR. You should have some way of claiming models from the list in [[i18n-JP] Translating `en/model_doc` docs to Japanese #27392](https://github.com/huggingface/transformers/issues/27392) to avoid future duplicate work.
> 2. This is also slightly out-of-date with the recent standardization to the model API docs introduced in [[Docs] Model_doc structure/clarity improvements #26876](https://github.com/huggingface/transformers/pull/26876). Would you please mind updating?
> 3. As you mentioned [here](https://github.com/huggingface/transformers/issues/27392#issuecomment-1803669747), try to work on ~10 at a time please. 🙏
> 4. You'll need to add the docs you're working on to the `toctree` to get the `build pr` CI test to pass.

I had already work on it weeks ago. so It slipped. Next PR would contain no more than 10 docs

File: docs/source/ja/model_doc/blenderbot.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+*オープンドメインのチャットボットの構築は、機械学習研究にとって難しい分野です。これまでの研究では次のことが示されていますが、
+ニューラル モデルをパラメーターの数とトレーニング対象のデータのサイズでスケーリングすると、結果が向上します。
+高性能のチャットボットには他の要素も重要であることを示します。良い会話には多くのことが必要です
+会話の専門家がシームレスに融合するスキル: 魅力的な話のポイントを提供し、話を聞く
+一貫した態度を維持しながら、知識、共感、個性を適切に表現する
+ペルソナ。適切なトレーニング データと選択が与えられた場合、大規模モデルがこれらのスキルを学習できることを示します。
+世代戦略。 90M、2.7B、9.4B パラメーター モデルを使用してこれらのレシピのバリアントを構築し、モデルを作成します。
+コードは公開されています。人間による評価では、当社の最良のモデルが既存のアプローチよりも優れていることがマルチターンで示されています
+魅力と人間性の測定という観点からの対話。次に、分析によってこの作業の限界について説明します。
+弊社機種の故障事例※
```
Comment (stevhliu) [inline docs/source/ja/model_doc/blenderbot.md:37]:
```suggestion
弊社機種の故障事例*
```

**Review by stevhliu — COMMENTED** [review]
Thanks for starting the `model/docs`! Please double-check the link formatting to ensure they aren't accidentally translated or broken :)

File: docs/source/ja/model_doc/blenderbot.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+>>> reply_ids = model.generate(**inputs)
+>>> print(tokenizer.batch_decode(reply_ids))
+["<s> That's unfortunate. Are they trying to lose weight or are they just trying to be healthier?</s>"]
+```
+
+## Documentation resources
+
+- [Causal language modeling task guide](../tasks/language_modeling)
+- [Translation task guide](../tasks/translation)
+- [Summarization task guide](../tasks/summarization)
```
Comment (stevhliu) [inline docs/source/ja/model_doc/blenderbot.md:75]: These are also untranslated

**Review by stevhliu — COMMENTED** [review]
One more untranslated section

File: docs/source/ja/model_doc/bert.md [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+
+このモデルは [thomwolf](https://huggingface.co/thomwolf) によって提供されました。元のコードは [こちら](https://github.com/google-research/bert) にあります。
+
+## Resources
+
+BERT を始めるのに役立つ公式 Hugging Face およびコミュニティ (🌎 で示される) リソースのリスト。ここに含めるリソースの送信に興味がある場合は、お気軽にプル リクエストを開いてください。審査させていただきます。リソースは、既存のリソースを複製するのではなく、何か新しいものを示すことが理想的です。
+
+<PipelineTag pipeline="text-classification"/>
+
+- [別の言語での BERT テキスト分類] に関するブログ投稿 (https://www.philschmid.de/bert-text-classification-in-a-different-language)。
```
Comment (stevhliu) [inline docs/source/ja/model_doc/bert.md:72]: Please check the links are formatted correctly for the BERT resources

```suggestion
- [別の言語での BERT テキスト分類](https://www.philschmid.de/bert-text-classification-in-a-different-language) に関するブログ投稿 。
```

File: docs/source/ja/model_doc/bert-generation.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+-->
+
+# BertGeneration
+
+## Overview
+
+BertGeneration モデルは、次を使用してシーケンス間のタスクに利用できる BERT モデルです。
+[シーケンス生成のための事前トレーニング済みチェックポイントの活用] で提案されている [`EncoderDecoderModel`]
+タスク(https://arxiv.org/abs/1907.12461)、Sascha Rothe、Sishi Nagayan、Aliaksei Severyn 著。
```
Comment (stevhliu) [inline docs/source/ja/model_doc/bert-generation.md:23]:
```suggestion
[シーケンス生成のための事前トレーニング済みチェックポイントの活用](https://arxiv.org/abs/1907.12461) で提案されている [`EncoderDecoderModel`]
タスク、Sascha Rothe、Sishi Nagayan、Aliaksei Severyn 著。
```

**Review by stevhliu — APPROVED** [review]
LGTM, thanks!

**Pull request merged**